cmd/admin-speedtest.go

```diff
@@ -82,14 +81,27 @@ EXAMPLES:
 type speedTestResult madmin.SpeedTestResult
 
 func (s speedTestResult) String() (msg string) {
-	msg += fmt.Sprintf("MinIO %s, %d servers, %d drives\n", s.Version, s.Servers, s.Disks)
+	for _, node := range s.PUTStats.Servers {
+		if node.Err != "" {
+			globalSpeedTestVerbose = true
+			break
+		}
+	}
+	for _, node := range s.GETStats.Servers {
+		if node.Err != "" {
+			globalSpeedTestVerbose = true
+			break
+		}
+	}
+
+	msg += fmt.Sprintf("\nMinIO %s, %d servers, %d drives\n", s.Version, s.Servers, s.Disks)
 	if globalSpeedTestVerbose {
 		msg += "\n"
 	}
-	msg += fmt.Sprintf("PUT: %s/s, %d objs/s\n", humanize.IBytes(s.PUTStats.ThroughputPerSec), s.PUTStats.ObjectsPerSec)
+	msg += fmt.Sprintf("PUT: %s/s, %s objs/s\n", humanize.Bytes(uint64(s.PUTStats.ThroughputPerSec)), humanize.Comma(int64(s.PUTStats.ObjectsPerSec)))
 	if globalSpeedTestVerbose {
 		for _, node := range s.PUTStats.Servers {
-			msg += fmt.Sprintf("   * %s:, %s/s, %d objs/s", node.Endpoint, humanize.IBytes(node.ThroughputPerSec), node.ObjectsPerSec)
+			msg += fmt.Sprintf("   * %s: %s/s %s objs/s", node.Endpoint, humanize.Bytes(uint64(node.ThroughputPerSec)), humanize.Comma(int64(node.ObjectsPerSec)))
```

Comment:
Are we not going to use IEC ? and use SI units instead? why?

Reply:
i see, since 1000 shows higher throughput than 1024 i had changed it to that, let me revert it